packages/appium/docs/en/guides/caps.md

```diff
@@ -67,6 +67,15 @@ Each Appium client has its own way of constructing capabilities and starting a s
 examples of doing this in each client library, head to the [Ecosystem](../ecosystem/index.md) page
 and click through to the appropriate client documentation.
 
+## BiDi
+
+Appium can support [WebDriver BiDi](https://w3c.github.io/webdriver-bidi/) protocol.
+The actual behavior depends on individual drivers.
+
+| Capability Name  | Type     | Description                                    |
+|------------------|----------|------------------------------------------------|
```

Comment:
I believe table definition can be simpler, we only need three dashes to define a row

Reply:
Yea, just followed existing ones for vanilla text. Simplified